static/js/config/routing.js

```diff
@@ -0,0 +1,37 @@
+'use strict'; //interpreted as script mode 
+
+//Think of app.js being here if it wasn't in a different file
+
+
+// configure our routes
+sortingApp.config(function($routeProvider) {
+
+    $routeProvider
```

Comment:
This was really what I was looking at the style guide for (because this was challenging for me to read initially). Here's an excerpt of how to do nested function calls (more than 2):
![screen shot 2017-03-13 at 11 00 51 pm](https://cloud.githubusercontent.com/assets/10744199/23886502/03ec09ca-0841-11e7-8d1e-2b6c66f462ef.png)